app/Filament/Server/Resources/FileResource/Pages/EditFiles.php

```diff
@@ -132,6 +133,9 @@ public function form(Form $form): Form
                                         ->getContent($this->path, config('panel.files.max_edit_size'));
                                 } catch (FileNotFoundException) {
                                     abort(404, $this->path . ' not found.');
+                                } catch (FileNotEditableException) {
+                                    redirect(ListFiles::getUrl());
+                                    $this->getFailureNotification($this->path);
```

Comment:
Remove function
```suggestion

                                    Notification::make()
                                       ->title('Could not edit!')
                                       ->body($this->path . ' is a directory.')
                                       ->danger()
                                       ->send();
```
_(no pint)_